### DISCUSSION TITLE
Organization-level custom domain not working

### DISCUSSION THREAD

**Discussion by aaronpk** [community/community#30136] [Other Feature Feedback, Questions, & Ideas]
I tried creating a custom domain for my organization to use with GitHub Pages. I wasn't able to find the actual documentation on how to do this, but a [previous discussion thread](https://github.com/orgs/community/discussions/23350) led me to the conclusion that I should create a repo in my org called `organization`, and set up that repo with a custom domain. That part is working fine.

https://drafts.oauth.net

I would expect now that the other repos using GitHub Pages would be accessible under this custom domain:

* Repo: https://github.com/oauth-wg/oauth-v2-1
* Original URL: https://oauth-wg.github.io/oauth-v2-1/
* Expected URL: https://drafts.oauth.net/oauth-v2-1/

However this doesn't seem to be working. What am I doing wrong?

**Comment by yoannchaudet** [comment]
Your organization repository should be named: `<org name>.github.io`. We talk about that here: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites.

So in your case, rename https://github.com/oauth-wg/organization to `oauth-wg.github.io`.

**Reply by aaronpk** [reply]
Thank you, that worked! 

May I suggest adding a link to that from the section I had been reading under "[Configure a custom domain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages)". 

There seems to be some missing docs to connect the dots here. Now that I know how it all works it makes sense.

**Reply by yoannchaudet** [reply]
Making a note for that!
